src/global/global.shared.scss

```diff
@@ -0,0 +1,40 @@
+//*****************************************************************************
+// Important
+//
+// Changes in global.scss and from global.scss referenced .scss files only
+// get compiled through a Stencil build. Restarting (stopping/starting) the build
+// is necessary so the changes get reflected in the global CSS (lyne-components.css)
+//
+// This is a known and unsolved stencil-sass bug: see
+// https://github.com/ionic-team/stencil-sass/issues/8
+// https://github.com/ionic-team/stencil-sass/issues/15
+//
+//*****************************************************************************
+
+// Core variables, are always needed
+
+// Does not work right now because CSS vars are scoped for :host in current
+// lyne-design-tokens, :host needs to be changed to
+// `%lyne-design-tokens-css-variables` in
+// node_modules/lyne-design-tokens/dist/scss/css_variables.scss so it can be
+// used within the given context e.g. :root but also :host depending where it
+// is used/referenced
+
+:root {
+  // Core variables for :root context
+  @include lyne-components-css-vars;
+}
+
+// Theming, override core variables and enable theme/mode through uncommenting
+// whished theme/mode
+// @import "theme/sbb-ch/mode--dark/variables.scss";
+// @import "theme/sbb-ch/mode--compact/variables.scss";
+@import "theme/sbb-ch/mode--dark-compact.variables.scss";
```

Comment:
This is just a proposal how we could define and handle themes and modes. The theme here would be `sbb-ch`, the modes would be `mode--compact` or `mode--dark` but it would also allow us to combine modes like `mode--dark-compact`. In my opinion the `compact` variant is more a mode than a actual theme but this seems to be quite thin line between a theme and a mode. Will search for some references how they are distingushed. 

Once we agree on the way themes are nested/constructed through variables the selection process could also be handled through the build process and generate multiple CSS files (for each defined variable file new output CSS) from which the user of the components decides which CSS file they wanna include/reference.